### PR TITLE
init for graalvm workshops - spring boot, quarkus, micronaut, helidon

### PR DESCRIPTION
**Pull request by paulparkinson**

### CODE REVIEW

File: graalvm-nativeimage/quarkus/src/main/resources/application.properties [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+quarkus.datasource.db-kind=oracle
+quarkus.datasource.jdbc.driver=oracle.jdbc.driver.OracleDriver
+quarkus.datasource.jdbc.url=jdbc:oracle:thin:@gdpaulsep27i_tp?TNS_ADMIN=/Users/pparkins/Downloads/Wallet_gdpaulsep27o
+quarkus.datasource.username=inventoryuser
+quarkus.datasource.password=Welcome12345
```
Comment (jasperan) [inline graalvm-nativeimage/quarkus/src/main/resources/application.properties:6]: Wallet password here

File: graalvm-nativeimage/quarkus/src/main/resources/application.properties [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+quarkus.datasource.db-kind=oracle
+quarkus.datasource.jdbc.driver=oracle.jdbc.driver.OracleDriver
+quarkus.datasource.jdbc.url=jdbc:oracle:thin:@gdpaulsep27i_tp?TNS_ADMIN=/Users/pparkins/Downloads/Wallet_gdpaulsep27o
```
Comment (jasperan) [inline graalvm-nativeimage/quarkus/src/main/resources/application.properties:4]: Can you pass that env variable like a HTTP parameter? I never knew that!

File: graalvm-nativeimage/quarkus/src/main/resources/microprofile-config.properties [hunk 3]
```diff
@@ -0,0 +1,9 @@
+## Copyright (c) 2021 Oracle and/or its affiliates.
+## Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/
+
+
+# DB properties
+oracle.ucp.jdbc.PoolDataSource.inventorypdb.URL = jdbc:oracle:thin:@gdpaulobi_tp?TNS_ADMIN=/Users/pparkins/Downloads/Wallet_gdpaulsep27o
+oracle.ucp.jdbc.PoolDataSource.inventorypdb.user = inventoryuser
+oracle.ucp.jdbc.PoolDataSource.inventorypdb.password = Welcome12345
```
Comment (jasperan) [inline graalvm-nativeimage/quarkus/src/main/resources/microprofile-config.properties:8]: password here

File: graalvm-nativeimage/springboot/src/main/java/com/oracle/jdbc/graalvm/GraalVMNativeImageJDBCDriver.java [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+	public static void main(String[] args) {
+		OracleDataSource ods;
+		try {
+			ods = new OracleDataSource();
+			//notice the servicename suffix appended, which can be _high, _low, ...
+//			ods.setURL("jdbc:oracle:thin:@${ATP Name}_high?TNS_ADMIN=/home/${MY_HOME_DIR}/myatpwallet");
+			ods.setURL("jdbc:oracle:thin:@ATP110345_high?TNS_ADMIN=/home/LL110345_U/myatpwallet");
+			ods.setUser("ADMIN");
+//			ods.setPassword("[ATP Admin Password]");
+			ods.setPassword("##03pRivvzR2q4jv");
```
Comment (jasperan) [inline graalvm-nativeimage/springboot/src/main/java/com/oracle/jdbc/graalvm/GraalVMNativeImageJDBCDriver.java:38]: There's also a password here but I'm not sure if it's a sensitive one or not

**Review by jasperan — CHANGES_REQUESTED** [review]

File: graalvm-nativeimage/helidon/src/main/resources/META-INF/microprofile-config.properties [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# Microprofile server properties
+server.port=8080
+server.host=0.0.0.0
+
+# DataSource properties
+javax.sql.DataSource.example.connectionFactoryClassName=oracle.jdbc.pool.OracleDataSource
+#javax.sql.DataSource.example.URL=jdbc:oracle:thin:@<ATP_NAME>_high?TNS_ADMIN=/home/<YOUR_USER>/<YOUR_WALLET_LOCATION>
+javax.sql.DataSource.example.URL=jdbc:oracle:thin:@ATP110345_high?TNS_ADMIN=/home/LL110345_U/myatpwallet
+javax.sql.DataSource.example.user=ADMIN
+javax.sql.DataSource.example.password=##03pRivvzR2q4jv
```
[the remark this answers is not in the document]
Comment (paulparkinson) [reply graalvm-nativeimage/helidon/src/main/resources/META-INF/microprofile-config.properties]: fixed it

File: graalvm-nativeimage/micronaut/src/main/resources/application.properties [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+flyway.datasources.default.enabled=true
+flyway.datasources.default.baseline-on-migrate=true
+flyway.datasources.default.baseline-version=1.0.0
+
+datasources.default.dialect=ORACLE
+#datasources.default.ocid=<YOUR_DB_OCID>
+datasources.default.username=ADMIN
+datasources.default.password=
+#datasources.default.password=<YOUR_PASSWORD>
+datasources.default.walletPassword=Welcome12345
```
[the remark this answers is not in the document]
Comment (paulparkinson) [reply graalvm-nativeimage/micronaut/src/main/resources/application.properties]: fixed it

File: graalvm-nativeimage/quarkus/src/main/resources/application.properties (hunk 1 above)
Comment (paulparkinson) [reply graalvm-nativeimage/quarkus/src/main/resources/application.properties:6]: fixed it (a lot of these are just templates/placeholders btw)

File: graalvm-nativeimage/quarkus/src/main/resources/application.properties (hunk 2 above)
Comment (paulparkinson) [reply graalvm-nativeimage/quarkus/src/main/resources/application.properties:4]: yep :)

File: graalvm-nativeimage/quarkus/src/main/resources/microprofile-config.properties (hunk 3 above)
Comment (paulparkinson) [reply graalvm-nativeimage/quarkus/src/main/resources/microprofile-config.properties:8]: fixed it

File: graalvm-nativeimage/springboot/src/main/java/com/oracle/jdbc/graalvm/GraalVMNativeImageJDBCDriver.java (hunk 4 above)
Comment (paulparkinson) [reply graalvm-nativeimage/springboot/src/main/java/com/oracle/jdbc/graalvm/GraalVMNativeImageJDBCDriver.java]: fixed it

**Review by jasperan — APPROVED** [review]

**Review by jasperan — APPROVED** [review]

**Review by jasperan — APPROVED** [review]

**Review by jasperan — APPROVED** [review]

**Review by jasperan — APPROVED** [review]

**Review by ttscoff — APPROVED** [review]

**Pull request merged**